modules/apps/liferay-theme/src/css/_custom.scss

```diff
@@ -0,0 +1,115 @@
+
+body {
+	font-family: Helvetica, Arial, sans-serif;
+	margin: 0;
+}
+
+.sign-in {
+	border: 2px solid #000;
+	border-radius: 200px;
+	color: #000;
+	display: block;
+	float: right;
+	margin-right: 15px;
+	padding: 6px;
+	text-transform: uppercase;
+}
+
+.header-banner {
+	margin-bottom: 80px;
+	padding-top: 90px;
```

Comment:
We should probably adjust the sizes based on the logged in/out state or mobile view, since there is a ton of white space at the top of the page when signed out.